Packs/SailPointIdentityNow/pack_metadata.json

```diff
@@ -0,0 +1,51 @@
+{
+    "name": "SailPointIdentityNow",
+    "description": "The SailPoint Identity Security platform can be configured either on-prem/single tenant SaaS, or multi-tenant. This package is intended to be used with the SaaS, multi-tenant solution, IdentityNow.",
```

Comment:
This doesn't explains what this pack does. I think we should change it like this:

"SailPoint Identity content pack allows you to utilize the deep contextual data and governance capabilities of SailPoint. It can be configured either on-prem/single tenant SaaS, or multi-tenant, and  is intended to be used with the SaaS, multi-tenant solution, IdentityNow"

Reply:
Updated with a better description.